### DISCUSSION TITLE
Update feature not working. Can I manually update?

### DISCUSSION THREAD

**Discussion by ikthrin** [wummle/arkos#13] [Q&A]
Hello, I have been using your WuMMLe fork of arkos on a rg351m since arkos was discontinued for this model. First of all, let me say Thank you for all your hard work! It is much appreciated. Anyway, like I said I have updated past arkos and onto wuMMlLe back when arkos was discontinued, and all has worked fine. The online update function no longer seems to work, however. My current version is the version that says it is 04052023 (which I am assuming is a date). I check for updates every month but have not received one, even though, according to the github page, there was a new version on August 26th.
When I run the updater, I type ok, then it just says "There was an error attempting this update" I have updated several times (wuMMLe) in the past with no problems. I even tried replacing the initial update script as per the Youtube video (even though I had of course done this before to get onto wummLe in the first place. This did not fix anything.
My Linux knowledge is limited, and I do not even have a machine that can view the Linux formatted portion of the card. The only was I can see anything on it is by using the file browser on the rg351. That said, I am able to view the log file, but there is nothing there telling what did or did not happen. It is an empty 0kb file, so sadly did not really help me.
So I was wondering if there is a was I could manually download the update and apply it by just copying files over somehow? (I can easily make a live image or virtual machine to read the card. I just cannot figure out what the problem could be to even look into, - I do have file management and command line experience going back to the DOS days, I am just not very current, I'm afraid. - so I have not done this yet. Again, thank you again for all your continued work. I would greatly appreciate any thoughts or assistance anyone might have as far as fixing this, or if there is someway manually update... Thank you for any assistance that anyone can provide. This one has been driving me crazy. Looking at the update script, in seems to download a zip file, which I manually downloaded, and then saved. I noticed it seems to call another script TheraUpdate.sh. When I look at this file in the home directory, it is another empty 0kb file, which does not seem right. I don't know if it is a file permission problem or what. It seems that could be related, but I have no idea how the permission got changed in the first place. I am able to delete the file, but when I copy the original over, it again turns into a 0kb empty file. Everything else seems to work just fine. My settings save and the device runs fine - it just will not update - driving me crazy. lol. Sorry for being so long winded. Again, thanks in advance for any help anyone can provide. Sorry to provide so much into, but here is one more bit. This is the log file of the most recent successful update. I don't see anything it did that would have prevented future updates, but again, most of this is over my head.
[update04052023.log](https://github.com/wummle/arkos/files/12915049/update04052023.log) Anyway, one more thank lol.

**Comment by wummle** [comment]
sounds like your linux partition maybe be full did you do an download all cores etc in retroarch ?
